designer/client/load-form-configurations.js

```diff
@@ -0,0 +1,28 @@
+function fetchConfigurations () {
+  return window.fetch('/configurations', {
+    method: 'get',
+    headers: {
+      Accept: 'application/json',
+      'Content-Type': 'application/json'
+    }
+  }).then(res => {
+    if (res.ok) {
+      return res.json()
+    } else {
+      throw res.error
+    }
+  })
+}
+
+async function loadConfigurations () {
+  return fetchConfigurations().then(data => {
+    return Object.values(data) || []
+  }).catch(error => {
+    console.log(error)
+    return []
+  })
+}
+
+export default {
+  loadConfigurations: loadConfigurations
```

Comment:
you can do just `default export  loadConfigurations`

Reply:
I'm not sure that I can stub it. E.G in form-details.test.js. There's nothing I can see documented in sinon about a way to do this. It looks like it's plausible using proxyquire but I think the use of that is grimmer than what we've currently got! Any thoughts?

Reply:
ah I see, np!